test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs

```diff
@@ -59,10 +59,33 @@ public void EnumeratorCacheIsScopedByAssembly()
             Assert.Equal(expectedCallCount, resourceAssembly2.GetManifestResourceStreamCallCount);
         }
 
+        [Fact]
+        public void ResourceManagerStringLocalizer_GetAllStrings_MissingResourceDoesntThrow()
```

Comment:
Should have a throw test-case. Xunit is awesome with them. `Assert.Throws` is your friend :smile: